plugins/ci/skills/fetch-tide-pr-status/fetch_tide_pr_status.py

```
@@ -0,0 +1,268 @@
+    job_config = presubmit_configs.get(branch)
+    if job_config:
+        required_jobs = [
+            j for j in jobs
+            if j["name"] in job_config and not job_config[j["name"]]["optional"]
+        ]
+    else:
+        required_jobs = [{"error": f"could not fetch presubmit config from openshift/release for {repo} branch {branch} — job classification unavailable"}]
+
+    blockers = []
+    if best:
+        blockers += [f"missing required label: {l['name']}" for l in best["labels"] if not l["have"]]
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)